--- ./lib/main.dart
+++ ./lib/main.dart
@@ -5,6 +5,7 @@ import 'package:flutter/material.dart';
 
 import 'package:flutter_localizations/flutter_localizations.dart';
 import 'package:flutter_web_plugins/url_strategy.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
 
 import 'auth/supabase_auth/supabase_user_provider.dart';
 import 'auth/supabase_auth/auth_util.dart';
@@ -16,25 +17,33 @@ import 'flutter_flow/internationalization.dart';
 import 'flutter_flow/nav/nav.dart';
 import 'index.dart';
 
-void main() async {
-  WidgetsFlutterBinding.ensureInitialized();
-  usePathUrlStrategy();
-
-  await SupaFlow.initialize();
-
-  await FlutterFlowTheme.initialize();
-
-  final appState = FFAppState(); // Initialize FFAppState
-  await appState.initializePersistedState();
-
-  // Start final custom actions code
-  await actions.initSupabaseRealtimeSubscriptions();
-  // End final custom actions code
-
-  runApp(ChangeNotifierProvider(
-    create: (context) => appState,
-    child: MyApp(),
-  ));
+Future<void> main() async {
+  await SentryFlutter.init((options) {
+    options.dsn =
+        'https://b378c6db98e5c54511fc29154355f563@o4506409083994112.ingest.sentry.io/4506409085042688';
+    // Set tracesSampleRate to 1.0 to capture 100% of transactions for performance monitoring.
+    // We recommend adjusting this value in production.
+    options.tracesSampleRate = 1.0;
+  }, appRunner: () async {
+    WidgetsFlutterBinding.ensureInitialized();
+    usePathUrlStrategy();
+
+    await SupaFlow.initialize();
+
+    await FlutterFlowTheme.initialize();
+
+    final appState = FFAppState(); // Initialize FFAppState
+    await appState.initializePersistedState();
+
+    // Start final custom actions code
+    await actions.initSupabaseRealtimeSubscriptions();
+    // End final custom actions code
+
+    return runApp(ChangeNotifierProvider(
+      create: (context) => appState,
+      child: MyApp(),
+    ));
+  });
 }
 

 class MyApp extends StatefulWidget {
--- ./pubspec.yaml
+++ ./pubspec.yaml
@@ -51,6 +51,7 @@ dependencies:
   provider: 6.0.5
   realtime_client: 1.4.0
   rxdart: 0.27.7
+  sentry_flutter: 7.13.2
   shared_preferences: 2.2.2
   shared_preferences_android: 2.2.1
   shared_preferences_foundation: 2.3.4