--- ./lib/login_page/login_page_widget.dart
+++ ./lib/login_page/login_page_widget.dart
@@ -520,9 +520,6 @@ class _LoginPageWidgetState extends State<LoginPageWidget>
                                                                 .passwordController
                                                                 .text,
                                                           );
-                                                          if (user == null) {
-                                                            return;
-                                                          }
 
                                                           if (loggedIn ==
                                                               true) {
@@ -1053,9 +1050,6 @@ class _LoginPageWidgetState extends State<LoginPageWidget>
                                                               .passwordCreateController
                                                               .text,
                                                         );
-                                                        if (user == null) {
-                                                          return;
-                                                        }
 
                                                         if (loggedIn) {
                                                           await Future.delayed(
