--- ./lib/login_page/login_page_widget.dart
+++ ./lib/login_page/login_page_widget.dart
@@ -687,10 +687,6 @@ class _LoginPageWidgetState extends State<LoginPageWidget>
                                                                           .passwordController
                                                                           .text,
                                                                     );
-                                                                    if (user ==
-                                                                        null) {
-                                                                      return;
-                                                                    }
 
                                                                     if (loggedIn ==
                                                                         true) {
@@ -1247,10 +1243,6 @@ class _LoginPageWidgetState extends State<LoginPageWidget>
                                                                         .passwordCreateController
                                                                         .text,
                                                                   );
-                                                                  if (user ==
-                                                                      null) {
-                                                                    return;
-                                                                  }
 
                                                                   if (loggedIn) {
                                                                     await Future.delayed(const Duration(
