--- ./lib/backend/api_requests/api_calls.dart
+++ ./lib/backend/api_requests/api_calls.dart
@@ -42,7 +42,7 @@ class GetCustomersAccountsCall {
   }) {
     return ApiManager.instance.makeApiCall(
       callName: 'Get Customers Accounts',
-      apiUrl: 'https://qecg6jbdmm.eu-west-1.awsapprunner.com/account',
+      apiUrl: 'http://localhost:8080/account',
       callType: ApiCallType.GET,
       headers: {
         'Content-Type': 'application/json',
@@ -127,7 +127,7 @@ class UpdateTopupPreferencesCall {
     return ApiManager.instance.makeApiCall(
       callName: 'Update Topup Preferences',
       apiUrl:
-          'https://qecg6jbdmm.eu-west-1.awsapprunner.com/wallet/${walletId}/topupPreference',
+          'http://localhost:8080/wallet/${walletId}/topupPreference',
       callType: ApiCallType.PUT,
       headers: {
         'Authorization': 'Bearer ${bearerToken}',
@@ -153,7 +153,7 @@ class MarkContractSignedCall {
     return ApiManager.instance.makeApiCall(
       callName: 'Mark Contract Signed',
       apiUrl:
-          'https://qecg6jbdmm.eu-west-1.awsapprunner.com/contract/${contractId}/sign',
+          'http://localhost:8080/contract/${contractId}/sign',
       callType: ApiCallType.PUT,
       headers: {
         'Authorization': 'Bearer ${bearerToken}',
@@ -175,7 +175,7 @@ class GetWalletsCall {
   }) {
     return ApiManager.instance.makeApiCall(
       callName: 'Get Wallets',
-      apiUrl: 'https://qecg6jbdmm.eu-west-1.awsapprunner.com/wallet',
+      apiUrl: 'http://localhost:8080/wallet',
       callType: ApiCallType.GET,
       headers: {
         'Content-Type': 'application/json',
@@ -197,7 +197,7 @@ class CustomerInviteLookupCall {
     return ApiManager.instance.makeApiCall(
       callName: 'Customer Invite Lookup',
       apiUrl:
-          'https://qecg6jbdmm.eu-west-1.awsapprunner.com/customer/invite/${inviteToken}',
+          'http://localhost:8080/customer/invite/${inviteToken}',
       callType: ApiCallType.GET,
       headers: {},
       params: {},
@@ -216,7 +216,7 @@ class ContractSigningEmbedCall {
     return ApiManager.instance.makeApiCall(
       callName: 'Contract Signing Embed',
       apiUrl:
-          'https://qecg6jbdmm.eu-west-1.awsapprunner.com/contract/signing-embed.html',
+          'http://localhost:8080/contract/signing-embed.html',
       callType: ApiCallType.GET,
       headers: {
         'Authorization': 'Bearer ${bearerToken}',
