--- ./lib/login_page/login_page_widget.dart
+++ ./lib/login_page/login_page_widget.dart
@@ -651,10 +651,6 @@ class _LoginPageWidgetState extends State<LoginPageWidget>
                                                                           .passwordController
                                                                           .text,
                                                                     );
-                                                                    if (user ==
-                                                                        null) {
-                                                                      return;
-                                                                    }
 
                                                                     if (loggedIn ==
                                                                         true) {
