*** ./lib/flutter_flow/flutter_flow_theme.dart	Tue Mar 19 09:12:20 2024
--- ./lib/flutter_flow/flutter_flow_theme.dart	Tue Mar 19 09:18:59 2024
***************
*** 2,7 ****
--- 2,9 ----
  
  import 'package:flutter/material.dart';
  import 'package:google_fonts/google_fonts.dart';
+ import 'package:my_energy/app_state.dart';
+ import 'package:my_energy/backend/schema/enums/enums.dart';
  
  import 'package:shared_preferences/shared_preferences.dart';
  
***************
*** 34,42 ****
  
    static FlutterFlowTheme of(BuildContext context) {
      deviceSize = getDeviceSize(context);
!     return Theme.of(context).brightness == Brightness.dark
!         ? DarkModeTheme()
!         : LightModeTheme();
    }
  
    @Deprecated('Use primary instead')
--- 36,51 ----
  
    static FlutterFlowTheme of(BuildContext context) {
      deviceSize = getDeviceSize(context);
!     bool isDarkMode = Theme.of(context).brightness == Brightness.dark;
!     EscoCodeEnum? esco = FFAppState().esco;
! 
!     if (esco == EscoCodeEnum.hmce) {
!       return isDarkMode ? HazelDarkModeTheme() : HazelLightModeTheme();
!     }
! 
!     // default theme is the wlce theme although we might think about a
!     // CEPRO fallback theme instead.
!     return isDarkMode ? DarkModeTheme() : LightModeTheme();
    }
  
    @Deprecated('Use primary instead')
